hack/testing-olm-upgrade/resources/cr.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: "logging.openshift.io/v1"
+kind: "Elasticsearch"
+metadata:
+  name: "elasticsearch"
+  annotations:
+      elasticsearch.openshift.io/loglevel: trace
+spec:
+  managementState: "Managed"
+  nodeSpec:
+    resources:
+      limits:
+        memory: 1Gi
+      requests:
+        cpu: 100m
+        memory: 1Gi
+  nodes:
+  - nodeCount: 3
+    roles:
+    - client
+    - data
+    - master
+    storage:
+      storageClassName: gp2
+      size: 100G
+  redundancyPolicy: SingleRedundancy
+  indexManagement:
+    policies:
+    - name: infra-policy
+      pollInterval: 1m
```

Comment:
for the sake of the OCP cluster resources, do we need these to run every minute?

Reply:
Fixed

Reply:
if we don't need to test the rollover stuff, why don't we make this closer to a `30m` poll

Reply:
Fixed, increased polling time to 30m